src/ResourceManager/Profile/AzureRM.Profile.Netcore.psd1

```diff
@@ -54,13 +54,13 @@ PowerShellVersion = '5.1'
 # RequiredModules = @()
 
 # Assemblies that must be loaded prior to importing this module
-# RequiredAssemblies = @()
+RequiredAssemblies = 'netcoreapp2.0\Microsoft.Azure.Commands.Common.Authentication.ResourceManager.dll'
```

Comment:
See the desktop version of thsi file - you will need to include the dependencies mentioned there (basically everything in your common targets file)